### PR TITLE
Add zoolz

### PR DESCRIPTION
**Pull request by Nicos18**

### CODE REVIEW

File: _data/backup.yml [hunk 1]
```diff
@@ -180,7 +188,8 @@ websites:
     tfa: No
     facebook: SugarSync
 
-  - name: Sync
+
+- name: Sync
```
Comment (mxxcon) [inline _data/backup.yml]: Please undo this change. This breaks YAML format

**Review by mxxcon — CHANGES_REQUESTED** [review]
Please add only 1 site per pull request. Either Degoo or Zoolz.
Whichever site you'll decide to keep in this pull request, please also add 32x32 logo file.

**Review by stephengroat — DISMISSED** [review]
thanks for the pr @Nicos18

**Review by mxxcon — CHANGES_REQUESTED** [review]
What about http://wiki.zoolz.com/how-do-i-enable-two-factor-authentication-on-my-zoolz-intelligent-account/ ?

**Comment by Nicos18** [comment]
@mxxcon unfortunately, this is only for business account, while for home users (http://ihome.zoolz.co.uk/), I didn't find anything about 2FA

For comparison, this is the "Account Settings" page for private users: we can change only the name & email and the password. That's it

cattura

**Comment by mxxcon** [comment]
Then perhaps it should be changed to `tfa: yes` and added exception field that it's only for biz accounts, not personal?

**Comment by Nicos18** [comment]
@mxxcon I've changed the file

**Review by mxxcon — DISMISSED** [review]

File: _data/backup.yml [hunk 2]
```diff
@@ -236,3 +236,12 @@ websites:
     tfa: Yes
     software: Yes
     doc: http://www.zetta.net/about/blog/how-set-zetta-two-factor-authentication/
+
+  - name: Zoolz
+    url: https://intelli.zoolz.com/
```
Comment (mxxcon) [inline _data/backup.yml]: I'm not sure we should use intelli subdomain. I think it's better to use www or just apex and let them redirect wherever they'll decide to...

File: _data/backup.yml [hunk 3]
```diff
@@ -231,3 +231,12 @@ websites:
...
     software: Yes
     doc: https://www.zetta.net/about/blog/how-set-zetta-two-factor-authentication/
+
+  - name: Zoolz
+    url: https://intelli.zoolz.com/
+    img: zoolz.png
+    tfa: Yes
+    software: Yes
+    exceptions:
+        text: "Only available for business accounts."
```
Comment (mxxcon) [inline _data/backup.yml:241]: Is it still restricted only to biz accounts?

Comment (Nicos18) [reply _data/backup.yml:241]: @mxxcon Yes, unfortunately

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Sorry for the delay @Nicos18. At least it's merged now 😅
